src/reference/asciidoc/jdbc.adoc

```diff
@@ -433,6 +433,8 @@ For this purpose, the `INT_CHANNEL_MESSAGE` has a `MESSAGE_PRIORITY` column to s
 In addition, a new `MESSAGE_SEQUENCE` column is also provided to achieve a robust first-in-first-out (FIFO) polling mechanism, even when multiple messages are stored with the same priority in the same millisecond.
 Messages are polled (selected) from the database with `order by MESSAGE_PRIORITY DESC NULLS LAST, CREATED_DATE, MESSAGE_SEQUENCE`.
 
+IMPORTANT: The `capacity` attribute is not allowed when the `message-store` attribute is used.
```

Comment:
Yes. I understand, but this your new sentence does not make sense in context of JDBC.
More over it looks like out-of-context when I read the whole text about `JdbcChannelMessageStore`. 
That's why I don't see reason in this sentence here. But I won't mind if it will be present in the `channel.adoc` for `PriorityChannel`.

Please, elaborate more if I am missing anything else.
Thanks